### DISCUSSION TITLE
Will Manifold keep on using Redis in the future

### DISCUSSION THREAD

**Discussion by dev-allinappli** [ManifoldScholar/manifold#3749] [Q&A]
(not exactly an issue, sorry for posting here but I didn't know where else to post)
As Redis recently moved to new licences (https://redis.io/blog/redis-adopts-dual-source-available-licensing/) which are not any more considered as really opensource, will Manifold keep using Redis or move to an alternative/fork like KeyDB (https://github.com/Snapchat/KeyDB)?
We're planing to install Manifold for a French institution and this part of the software is important to us.

**Comment by zdavis** [comment]
As of v9, Manifold no longer uses Redis, which was mainly used for storing caching and job storage. That's all been moved to the PG database at this point.
